### PR TITLE
Model tests

### LINKED ISSUE #13: Add tests: models
Add tests for the models.

### PR DESCRIPTION
**Pull request by somexpert**
Adds tests for the models. Will fix #13.

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @damonkelley, this is ready for review.

File: tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
+from datetime import datetime, timedelta
+
+from django.conf import settings
+
+import pytest
+
+from .factories import (ValueFactory, ValuesetFactory, MetadataFactory,
```
Comment (damonkelley) [inline tests/test_models.py]: This is an _import smell_ to me :stuck_out_tongue_closed_eyes: 

Probably better of with 

```python 
from . import factories
```

Comment (somexpert) [inline tests/test_models.py]: It _is_ a lot of imports. Alright, I'll make the switch.

File: tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+
+class TestProject:
+    def test_unicode(self):
+        project = ProjectFactory.build()
+
+        assert unicode(project) == project.project_slug
+
+    def test_nomination_message_before_nomination_window(self):
+        project = ProjectFactory.build(
```
Comment (damonkelley) [inline tests/test_models.py]: This is being duplicated in almost every test in the test case. Have you considered moving it into a fixture?

Comment (damonkelley) [inline tests/test_models.py]: ah, I see that they are **not** all the same. NVM

**Comment by ldko** [comment]
:+1:

**Comment by damonkelley** [comment]
:+1: 

But I still don't agree with the duplication in `TestURL`, FYI  :)

**Comment by somexpert** [comment]
@damon, noted. I'm going to merge this in, but I may get to revisit this later once I'm working on updates/modifications.

**Pull request merged**